compute_tools/src/compute.rs

```diff
@@ -384,30 +384,29 @@ impl ComputeNode {
         let mut client = config.connect(NoTls)?;
         let pageserver_connect_micros = start_time.elapsed().as_micros() as u64;
 
+        let replica = if spec.spec.mode != ComputeMode::Primary {
```

Comment:
I'd suggest building an array of arguments here. Something like:
```
let mut args = Vec::new();

if spec.spec.mode != ComputeMode::Primary {
    args.append("--replica");
}
..
```

Reply:
Thank you!
Done.